vignettes/custom_appearance.Rmd

```diff
@@ -90,3 +94,167 @@ lyt3 <- basic_table() %>%
 tbl3 <- build_table(lyt3, ex_adsl)
 tbl3
 ```
+
+### Top-left Materials
+
+The sequence of sequence of strings printed in the area between the column header display and the first row label (\emph{top left material}) can be modified using the `top_left()` function.
+
+```{r}
+lyt <- basic_table() %>%
+  split_cols_by("ARM") %>%
+  split_rows_by("SEX") %>%
+  analyze(c("AGE", "STRATA1"), function(x) {
+    if (is.numeric(x)) {
+      in_rows(
+        "mean" = rcell(mean(x)),
+        "sd" = rcell(sd(x)),
+        .formats = c("xx.x"), .aligns = "left"
+      )
+    } else if (is.factor(x)) {
+      rcell(length(unique(x)), align = "right")
+    } else {
+      stop("Unsupported type")
+    }
+  }, show_labels = "visible", na_str = "NE") %>%
+  build_table(ex_adsl)
+
+lyt
+
+# Adding top-left material
+top_left(lyt) <- "New top-left material here"
+
+lyt
+```
+
+
+### Table Inset
+
+Table title, table body, referential footnotes and and main footers can be inset from the left alignment of the titles and provenance footer materials.
+
+Below is a simplified example:
+
+```{r}
+lyt <- basic_table() %>%
```

Comment:
As above, primary way of setting table_inset is inside `basic_table`, please document that in addition to the post processing you already have